jgrapht-core/src/main/java/org/jgrapht/graph/BaseIntrusiveEdgesSpecifics.java

```diff
@@ -148,11 +148,40 @@ public void setEdgeWeight(E e, double weight)
      */
     public abstract boolean add(E e, V sourceVertex, V targetVertex);
 
+    boolean addIntrusiveEdge(E edge, V sourceVertex, V targetVertex, IE e)
```

Comment:
This needs to be protected. A few of our users have their own specifics implementation and need access here.

Reply:
I changed the modifier to `protected`.